packages/ckb-sdk-rpc/src/defaultRPC.ts

```diff
@@ -273,9 +273,15 @@ export class DefaultRPC {
    * @description rpc to send a new transaction into transaction pool
    * @param {object} rawTransaction - a raw transaction includes cell deps, inputs, outputs, version, and witnesses,
    *                                  detailed info could be found in ckb-types
+   * @param {string} [outputsValidator] - Validates the transaction outputs before entering the tx-pool,
+   *                                  an optional string parameter (enum: default | passthrough ),
+   *                                  null means using default validator, passthrough means skipping outputs validation
    * @return {Promise<string>} transaction hash
    */
-  public sendTransaction!: (tx: CKBComponents.RawTransaction) => Promise<CKBComponents.Hash>
+  public sendTransaction!: (
+    tx: CKBComponents.RawTransaction,
+    outputsValidator: CKBComponents.OutputsValidator,
```

Comment:
According rpc document, outputsValidator can be null. So `outputsValidator: CKBComponents.OutputsValidator` will be more better ?

Reply:
I think I'm using `outputsValidator: CKBComponents.OutputsValidator`

Reply:
Get